pandas/tests/groupby/test_categorical.py

```diff
@@ -67,6 +68,12 @@ def setUp(self):
              'E': np.random.randn(11),
              'F': np.random.randn(11)})
 
+    def test_level_groupby_get_group(self):
+        # gh15155
+        testGroupBy = self.mframe.groupby(level=["first"])
+        assert_numpy_array_equal(testGroupBy.get_group("foo").values,
```

Comment:
That still fails (now using the code from the example to generate the test data):
```
# assert_frame_equal(testGroupedBy.get_group("a"),test.loc[["a"]])
MultiIndex level [0] classes are not equivalent
[left]:  CategoricalIndex([u'a', u'a', u'a', u'a', u'a'], categories=[u'a', u'b'], ordered=False, name=u'Index1', dtype='category')
[right]: Index([u'a', u'a', u'a', u'a', u'a'], dtype='object', name=u'Index1')

```